### PR TITLE
Update Thoth configuration file and Thoth's s2i configuration

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

File: .thoth.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+host: khemenu.thoth-station.ninja
+tls_verify: false
+requirements_format: pipenv
+
+runtime_environments:
+  - name: 'ubi:8'
```
Comment (harshad16) [inline .thoth.yaml]: shouldn't this be changed to `rhel:8.0`?

Comment (harshad16) [reply .thoth.yaml]: @fridex

Comment (fridex) [reply .thoth.yaml]: Oh, yes... I'll adjust PRs. Thanks for spotting this :+1:

Comment (harshad16) [reply .thoth.yaml]: thanks for raising all the PR's

**Pull request merged**